atom/browser/atom_download_manager_delegate.cc

```diff
@@ -92,6 +100,11 @@ bool AtomDownloadManagerDelegate::DetermineDownloadTarget(
     const content::DownloadTargetCallback& callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
+  AtomBrowserContext* browser_context = static_cast<AtomBrowserContext*>(
+      download_manager_->GetBrowserContext());
+  default_download_path_ = browser_context->prefs()->GetFilePath(
```

Comment:
Since we are reading `kDownloadDefaultDirectory` every time, I think we no longer needs to keep `default_download_path_` as a member data of the class.